.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Found a bug in Fluent UI React-Native? Please let us know.
+---
+
+<!--
+Thanks for contacting us! We're here to help.
+Before you report an issue, check if it's been reported before:
+  * Search: https://github.com/microsoft/fluentui-react-native/search?type=Issues
+Note that if you do not provide enough information to reproduce the issue, we may not be able to take action on your report.
+-->
+
+### Environment Information
+
+- **Platform**:
+  - [ ] iOS
+  - [ ] macOS
+  - [ ] win32
+  - [ ] windows
+- **Package version(s)**: (fill this out, include which package manager you're using)
```

Comment:
Are there commands we can have them run and just copy the output (node -v, etc.)

Reply:
Seems unnecessary? I'd imagine people already know how to do that